deploy/packaging/rpm/centos/6/SOURCES/geowave-init.sh

```diff
@@ -49,7 +49,10 @@ wait_for_start() {
 start() {
   echo -n "Starting $DISPLAY_NAME: "
   if ! is_running; then
-  	runuser -l geowave -c "$GEOWAVE_HOME/geoserver/bin/startup.sh &" > /dev/null 2>&1
+    if [ -f /etc/geowave/geowave.config ]; then
+      SOURCE_CMD="source /etc/geowave/geowave.config;"
```

Comment:
Does this file get manage via alternatives?
(I can see arguments for different config files per version, but then I'm not sure what the expectation is re: upgrading - copy over currently linked file?)